lib/github.js

```diff
@@ -71,6 +71,12 @@ Organization.prototype.getRepositories = function(callback) {
             cli.debug('next page ' + this.nextPageUrl);
             cli.debug('last page ' + this.getLastPage());
 
+            if ( body.message ) {
```

Comment:
Understood. This is awkward, the request library should be calling the callback with the error right away, without the need for us to check the status code. I'll see if I can find anything on their repository.